LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs

```diff
@@ -293,6 +293,8 @@ public static Model GetModel(string name)
                     return Model.Z390_UD;
                 case var _ when name.Equals("Z690 AORUS PRO", StringComparison.OrdinalIgnoreCase):
                     return Model.Z690_AORUS_PRO;
+                case var _ when name.Equals("Z690 GAMING X DDR4", StringComparison.OrdinalIgnoreCase):
+                    return Model.Z690_GAMING_X;
```

Comment:
Should I break this out into DDR4 vs DDR5? My _guess_ is that both boards are probably similar enough in ways other than their RAM, but I can't tell for sure.

Reply:
If you can't verify, don't put it in.

Reply:
OK, renamed to explicit DDR4. I suppose if someone wants support for the DDR5 edition of the board later, then can volunteer to test whether it's the same or whether it needs a duplicate-and-tweaked entry.